app/components/concept-page/question-card.ts

```diff
@@ -16,11 +16,16 @@ interface Signature {
 
 export default class QuestionCardComponent extends Component<Signature> {
   @tracked selectedOptionIndex: number | null = null;
+  @tracked _hasShownConfetti = false;
```

Comment:
```suggestion
  @tracked hasShownConfetti = false;
```

We don't need a separate underscored version